### PR TITLE
fix(storefront): BCTHEME-319 Write a review modal extra executions

### PR DESCRIPTION
**Pull request by yurytut1993**
#### What?

This PR has update for Review Form Modal on PDP and helps to avoid unnecessary executions of this modal

#### Tickets / Documentation

[Ticket](https://jira.bigcommerce.com/browse/BCTHEME-319)

### CODE REVIEW

File: assets/js/theme/product.js [hunk 1]
```diff
@@ -46,7 +46,11 @@ export default class Product extends PageManager {
 
         const review = new Review($reviewForm);
 
-        $(document).on('opened.fndtn.reveal', () => this.reviewModal.setupFocusableElements(WRITE_REVIEW));
+        $(document).on('opened.fndtn.reveal', (event) => {
+            if ($(event.target).attr('id') === 'modal-review-form') {
+                this.reviewModal.setupFocusableElements(WRITE_REVIEW);
+            }
+        });
```
Comment (BC-tymurbiedukhin) [inline assets/js/theme/product.js:53]: Could we use selector argument for .on jquery method? See https://api.jquery.com/on/

Comment (yurytut1993) [reply assets/js/theme/product.js:53]: Can you please give an example? Do you mean like $( "body" )?

Comment (BC-tymurbiedukhin) [reply assets/js/theme/product.js:53]: `$( "#dataTable tbody" ).on( "click", "tr", function() {
  console.log( $( this ).text() );
});` from the attached link

Comment (yurytut1993) [reply assets/js/theme/product.js]: Done

**Review by BC-tymurbiedukhin — APPROVED** [review]
LGTM

**Review by bc-alexsaiannyi — APPROVED** [review]

**Pull request merged**